mmdet/core/anchor/anchor_target.py

```diff
@@ -125,7 +136,8 @@ def anchor_target_single(flat_anchors,
     num_valid_anchors = anchors.shape[0]
     bbox_targets = torch.zeros_like(anchors)
     bbox_weights = torch.zeros_like(anchors)
-    labels = anchors.new_zeros(num_valid_anchors, dtype=torch.long)
+    labels = anchors.new_zeros(
+        num_valid_anchors, dtype=torch.long) + background_label
```

Comment:
We could change it to `labels = anchors.new_empty(num_valid_anchors, dtype=torch.long).fill_(background_label)`.
Just to be consistent with `bbox_target`

Reply:
sure